polkadot/runtime/rococo/src/xcm_config.rs

```diff
@@ -188,7 +194,8 @@ impl xcm_executor::Config for XcmConfig {
 	type SubscriptionService = XcmPallet;
 	type PalletInstancesInfo = AllPalletsWithSystem;
 	type MaxAssetsIntoHolding = MaxAssetsIntoHolding;
-	type FeeManager = ();
+	type FeeManager =
+		XcmFeesToAccount<Self, (SystemParachains, HereLocation), AccountId, TreasuryAccount>;
```

Comment:
@franciscoaguirre I added `HereLocation` here because most of the `pallet_xcm` benchmarks previously used `new_executor::<T>(Default::default())` which resulted in `new_executor::<T>(MultiLocation { parents: 0, interior: Junctions::Here })`, but I see now that `T::DeliveryHelper::ensure_successful_delivery` is used in the relevant benchmarks, so maybe we could remove this `HereLocation` and/or maybe put it behind `#[cfg(feature = "runtime-benchmarks")]`?

I am also not sure if `Here` can be used as real origin for any processing in `XcmExecutor`, I am not aware of any scenario, but maybe there is some and it is possible. So the consequence could be like: if we dont add `Here` to waived location, then `AssetTransactor::deposit_asset` or `AssetTransactor::withdraw_asset` would probably fail because we dont have `LocationToAccountId` converter for `Here` everywhere.

There is a possiblity to add converter for `Here` with [`DescribeTerminus`](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/xcm-builder/src/location_conversion.rs#L46-L50), but again this is added just to [`AssetHubKusama`](https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs#L82) or [`AssetHubPolkadot`](https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs#L72)  and neither `BridgeHubKusama` nor `BridgeHubPolkadot` nor `Collectives-polkadot` nor relaychain [`Kusama`](https://github.com/polkadot-fellows/runtimes/blob/main/relay/kusama/src/xcm_config.rs#L68-L73) nor [`Polkadot`](https://github.com/polkadot-fellows/runtimes/blob/main/relay/polkadot/src/xcm_config.rs#L68C10-L73) has it. And also if we add `DescribeTerminus` for `Here` then we probably need to think of existential deposit for this account.

Reply:
We could make `Here` waived. Then if you execute something locally **on behalf of the chain**, you don't need to pay for delivery fees.
I don't think we have anything like that now, so we could add it when we need to.
Also, this is a very simple configuration to add to the test runtimes first and then to the production runtimes if we deem it necessary.

I'll remove it for now

Reply:
> There is a possiblity to add converter for `Here` with [`DescribeTerminus`](https://github.com/paritytech/polkadot-sdk/blob 
... and neither `BridgeHubKusama` nor `BridgeHubPolkadot` nor `Collectives-polkadot` nor relaychain [`Kusama`](https://github.com/polkadot-fellows/runtimes/blob/main/relay/kusama/src/xcm_config.rs#L68-L73) nor [`Polkadot`](https://github.com/polkadot-fellows/runtimes/blob/main/relay/polkadot/src/xcm_config.rs#L68C10-L73) has it. And also if we add `DescribeTerminus` for `Here` then we probably need to think of existential deposit for this account.

I have no idea what kind of coincidence is this or whatever, but somebody just opened [PR](https://github.com/paritytech/polkadot-sdk/pull/1928) exactly with adding `DescribeFamily<DescribeAllTerminal>>` including `DescribeTerminus` to the system parachain runtimes few minutes ago :D :D :D

Reply:
That looks good, we can bring back `Here` after this is merged 😄